sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md

```diff
@@ -1,12 +1,31 @@
 # Release History
 
-## 1.0.0 (Unreleased)
+## 1.0.0 (2020-10-30)
+
+### New Features
 
 - Regenerate protocol layer from service API version 2020-10-31
 - Update service API version to use service API version 2020-10-31 by default
-- Add optional parameters for TraceParent and TraceState to all service request APIs to support distributed tracing
+
+
+### Breaking changes
+
+- Replace all Response<string> and Pageable<string> APIs with Response<T> and Pageable<T> respectively
+- Rename CreateDigitalTwin, CreateRelationship and CreateEventRoute APIs to CreateOrReplaceDigitalTwin, CreateOrReplaceRelationship and CreateOrReplaceEventRoute respectively
 - Renamed model type "ModelData" to "DigitalTwinsModelData" to make type less generic, and less likely to conflict with other libraries
-- Renamed model type "RequestOptions" to "DigitalTwinsRequestOptions" to make type less generic, and less likely to conflict with other libraries
+- Renamed model type "EventRoute" to "DigitalTwinsEventRoute" to make type less generic, and less likely to conflict with other libraries
+- Remove UpdateOperationsUtility and replace it with a direct dependency on JsonPatchDocument from Azure.Core
+- Remove WritableProperty since service no longer returns that type
+- Remove MaxItemCount parameter as an option for GetEventRoutes APIs since users are expected to provide page size in pageable type's .AsPages() method instead
+- Rename DigitalTwinsModelData field "DisplayName" to "LanguageDisplayNames" for clarity
+- Rename DigitalTwinsModelData field "Description" to "LanguageDescriptions" for clarity
```

Comment:
GetModels now takes the parameters in the options object instead of as straight up parameters